src/stackbox/slice/mod.rs

```diff
@@ -43,6 +51,32 @@ impl<'frame, Item: 'frame> StackBox<'frame, [Item]> {
         Some(hd.assert_singleton().into_inner())
     }
 
+    /// [`VecDeque`](alloc::collections::VecDeque)-like behavior for [`StackBox`]: pop its last item.
+    ///
+    /// ```
+    /// use stackbox::Slot;
+    /// let mut slot = Slot::VACANT;
+    /// let arr = slot.stackbox([0, 1, 2]);
+    /// let mut slice = arr.into_slice();
+    /// assert_eq!(slice.pop_back(), Some(2));
+    /// ```
+    pub fn pop_back(self: &'_ mut StackBox<'frame, [Item]>) -> Option<Item> {
+        if self.is_empty() {
+            return None;
+        }
+        let len = self.len();
+        let this = core::mem::take(self);
+        let (hd, tl) = this.stackbox_split_at(len - 1);
+        *self = hd;
+        Some(tl.assert_singleton().into_inner())
+    }
+
+    #[deprecated]
+    /// Use [`pop_front`](StackBox::pop_front) instead
```

Comment:
If we are to keep the deprecated function, then the link must use `[]` for it to work with rustdoc. We might as well adjust the `deprecated` message itself to include the suggestion 🙂:

```suggestion
    #[deprecated = "Use `.pop_front()` instead"]
    /// Deprecated alias for [`.pop_front()`][Self::pop_front].
```

That being said, I think we can go with a "major" bump (`0.2.0`), given the age of the crate, and skip the deprecated cruft